README.md

```diff
@@ -67,5 +95,6 @@ async def main():
 asyncio.get_event_loop().run_until_complete(main())
 ```
 
+
 Please do note that this example requires python 3.5+ for the `async`/`await`
 syntax. nio on the other hand works with older python versions as well.
```

Comment:
This should be moved to the beginning of the usage section (after "Unless special requirements...") and say "these examples".